build/test.ps1

```diff
@@ -38,7 +38,17 @@ function Test-One {
     }
 }
 
+function Test-VSCode {
+    Param([string] $path);
+
+    Write-Host "##[info]Testing VSCode Extension inside $path..."    
+    Push-Location (Join-Path $PSScriptRoot $path)
+        npm run unittest
+    Pop-Location
+}
```

Comment:
You'll need to have something like this to make sure the failed tests get reported as expected.
```suggestion
    if  ($LastExitCode -ne 0) {
        Write-Host "##vso[task.logissue type=error;]Failed to test VSCodeExtension at $path."
        $script:all_ok = $False
    }
}
```